File: Python/Dem_tu.py
def demTu(tu):
    tu = list(tu)
    doDai = len(tu)
    count = int(0)
    print(doDai)
    for i in range(0,doDai):
        for j in range(i+1,doDai):
            if(tu[i]=='.'):
                if(tu[j]!='.'):
                    count+=1
    if(tu[:1]=="."):
        if(tu[:-1]=="."):
            count-=1
    if(tu[:1]!="."):
        if(tu[:-1]!="."):
            count+=1
    print(count)

tu = str(input())
demTu(tu)

File: C_C++/Dem_tu.cpp
#include <iostream>
#include <array>
using namespace std;

void demTu(string &tu)
{
	cin >> tu;
	int doDai = tu.length();
	int i, count = 0;
	for(i=0;i<doDai;i++)
	{
	 	if(tu[i]=='.' && tu[i+1]!='.')
	 	{
	 		count++;
		}	
	}
	if(tu.back()=='.' && tu.front()=='.')
		count--;
	if(tu.back()!='.' && tu.front()!='.')
		count++;
	cout << count;
}

int main()
{
	string tu;
	demTu(tu);
}